packages/haul-core/src/config/types.ts

```diff
@@ -47,6 +47,11 @@ export type BundleConfig = {
   >;
   hasteOptions?: any;
   transform?: WebpackConfigTransform;
+  /* external bundles */
```

Comment:
instead of a comment, how about creating a new type name `ExternalBundleConfig` and spread/join it here